README.md

```diff
@@ -262,6 +262,49 @@ gulp restore \
 ```
 (Note: The restore time is a JavaScript timestamp, in milliseconds.)
 
+## Running locally
+
+To run this locally (such as for development)...
+
+1. Open a terminal to **THIS** repo's root folder and run the following:
+   - `make dynamodb-tables` (NOTE: You may need to run this twice.)
```

Comment:
How would I know if a second run is needed? Is it idempotent so I could run it any number of times without a problem?